delivery_ups_oca/README.rst

```diff
@@ -1,32 +1,34 @@
-======================
-Carrier labels for ups
-======================
+================
+Delivery UPS OCA
+================
 
 .. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
    !! This file is generated by oca-gen-addon-readme !!
    !! changes will be overwritten.                   !!
    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+.. |badge1| image:: https://img.shields.io/badge/maturity-Production%2FStable-green.png
     :target: https://odoo-community.org/page/development-status
-    :alt: Beta
+    :alt: Production/Stable
 .. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fdelivery--carrier-lightgray.png?logo=github
-    :target: https://github.com/OCA/delivery-carrier/tree/12.0/delivery_carrier_label_ups
+    :target: https://github.com/OCA/delivery-carrier/tree/13.0/delivery_ups_oca
     :alt: OCA/delivery-carrier
 .. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
-    :target: https://translation.odoo-community.org/projects/delivery-carrier-12-0/delivery-carrier-12-0-delivery_carrier_label_ups
+    :target: https://translation.odoo-community.org/projects/delivery-carrier-13-0/delivery-carrier-13-0-delivery_ups_oca
     :alt: Translate me on Weblate
 .. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
-    :target: https://runbot.odoo-community.org/runbot/99/12.0
+    :target: https://runbot.odoo-community.org/runbot/99/13.0
     :alt: Try me on Runbot
 
 |badge1| |badge2| |badge3| |badge4| |badge5| 
 
 This module adds `UPS <https://ups.com>`_ to the available carriers.
 
+Register shipping, generate label, read shipping state and cancel shippment by webservices.
```

Comment:
Remember to split lines with the same 80 cols criteria on text files for better diff reading:

```suggestion
It allows you to register shippings, generate labels, read shipping states and 
cancel shipments using UPS webservice, so no need of exchanging any kind of file.
```